lib/riot-api/map-name-to-ids.js

```diff
@@ -0,0 +1,59 @@
+"use strict";
+var async = require('async');
+var riotRequest = require('./request');
+var duration = require('../helper/duration');
+
+// Create cargo limit
+var API_LIMIT = 40;
+module.exports = function mapNameToIds(region, summoners, callback) {
+  var results = [];
+
+  var cargo = async.cargo(function(summonerNames, cb) {
+
+    var endpoint = '/api/lol/' + region.toLowerCase() + '/v1.4/summoner/by-name/' + encodeURIComponent(summonerNames.join(','));
+
+    riotRequest(region, endpoint, duration.A_DAY, function(err, data) {
```

Comment:
there is no real reason to cache this very particular query, since it's very unlikely to ever be done again